sorting/insert_sort.py

```diff
@@ -0,0 +1,24 @@
+"""Insert Sort Implementation."""
+
+
+def insert_sort(lst=[]):
```

Comment:
Why does this function have a  kwarg of `lst`? This shouldn't start with a default value.